packages/component/src/Localization/ja-JP.js

```diff
@@ -37,6 +37,14 @@ function xMinutesAgo(dateStr) {
   });
 }
 
+function botSaidSomething(avatarInitials, text) {
+  return `ボット${avatarInitials}は${text}と言いました`;
```

Comment:
Do we want to add spaces between Japanese and English? If yes, please add. If no, please remove it from line 45.

Reply:
Nope, spaces aren't used in JP